pkg/driver/driver.go

```diff
@@ -91,14 +91,6 @@ func NewDriver(endpoint string, mpVersion string, nodeID string) (*Driver, error
 }
 
 func (d *Driver) Run() error {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-	tokenFile := os.Getenv(webIdentityTokenEnv)
-	if tokenFile != "" {
-		klog.Infof("Found AWS_WEB_IDENTITY_TOKEN_FILE, syncing token")
-		go tokenFileTender(ctx, tokenFile, "/csi/token")
-	}
```

Comment:
This logic has been moved to [credentials_sts_web_identity.go](https://github.com/awslabs/mountpoint-s3-csi-driver/blob/4c7f236427571c23a46ac50b49149e714c9cdcc7/pkg/driver/node/credentialprovider/credentials_sts_web_identity.go). Since we enabled [`requiresRepublish`](https://github.com/awslabs/mountpoint-s3-csi-driver/blob/main/charts/aws-mountpoint-s3-csi-driver/templates/csidriver.yaml#L15C3-L15C26) as part of Pod-level identity support, Kubernetes will call `NodePublishVolume` periodically to update existing service account tokens before they expire, and the credential provider will be called as part of this method. Another reason for this change is that, this assumes a single location for service account tokens for Driver-level identity, but with containerization this won't be the case. See the note regarding credential provider in the original PR description for more context.